peppercorn/tests.py

```diff
@@ -56,53 +44,66 @@ def content_type_and_body(fields):
     lines.append('--' + BOUNDARY + '--')
     lines.append('')
 
-    body = CRLF.join(lines)
-    content_type = 'multipart/form-data; boundary=%s' % BOUNDARY
-
-    return content_type, body
+    return CRLF.join(lines)
 
 
 @pytest.fixture(scope="function")
-def content_type(content_type_and_body):
-    content_type, _ = content_type_and_body
-    return content_type
+def body_bytesio(body):
+    return BytesIO(body.encode("utf-8"))
 
 
 @pytest.fixture(scope="function")
-def body(content_type_and_body):
-    _, body = content_type_and_body
-    return body
+def content_type():  # pragma NO COVER Python >= 3.13
+    return 'multipart/form-data; boundary=%s' % BOUNDARY
 
 
 @pytest.fixture(scope="function")
-def environ_for_mpfs(environ, content_type):
-    environ["CONTENT_TYPE"] = content_type
-    environ["REQUEST_METHOD"] = "POST"
-    return environ
-
+def wsgi_environ():  # pragma NO COVER Python >= 3.13
+    return {
+        "wsgi.url_scheme": "http",
+        "SERVER_NAME": "localhost",
+        "SERVER_PORT": "8080",
+        "REQUEST_METHOD":"POST",
+        "PATH_INFO": "/",
+        "QUERY_STRING":"",
+    }
 
 @pytest.fixture(scope="function")
-def body_bytesio(body):
-    return BytesIO(body.encode("utf-8"))
+def cgi_environ(wsgi_environ, content_type):  # pragma NO COVER Python >= 3.13
+    wsgi_environ["CONTENT_TYPE"] = content_type
+    wsgi_environ["REQUEST_METHOD"] = "POST"
+    return wsgi_environ
 
 
 @pytest.fixture(scope="function")
-def headers(content_type, body):
+def cgi_headers(content_type, body):  # pragma NO COVER Python >= 3.13
     return {
         "content-length": str(len(body)),
         "content-type": content_type,
     }
 
+
 @pytest.fixture(scope="function")
-def multipart_field_storage(body_bytesio, environ_for_mpfs, headers):
+def cgi_fieldstorage(
+    body_bytesio, cgi_environ, cgi_headers
+):  # pragma NO COVER Python >= 3.13
+    from cgi import FieldStorage
+
     return FieldStorage(
         fp=body_bytesio,
-        environ=environ_for_mpfs,
+        environ=cgi_environ,
         keep_blank_values=1,
         headers=headers,
```

Comment:
I think this should be ``headers=cgi_headers``, the tests are failing because of this.